.github/styles/ttd-light/So.yml

```diff
@@ -1,7 +1,7 @@
 ---
 extends: existence
 message: "Don't start a sentence with '%s'"
-ignorecase: true
+ignorecase: false
```

Comment:
@janisz what is this change doing?

Reply:
the regexp `'(?:[;-]\s)so[\s,]|So[\s,]'` is case sensitive. If we ignore case we will have false positives 
see https://regexr.com/5b237

Reply:
ok gotcha.